background/background.js

```diff
@@ -43,9 +43,11 @@ Find.register("Background", function(self) {
             Find.browser.tabs.query({}, (tabs) => {
                 for(let tabIndex = 0; tabIndex < tabs.length; tabIndex++) {
                     let url = tabs[tabIndex].url;
-                    if(url.match(/chrome:\/\/.*/)
+                    if(url.match(/(?:chrome(?:-extension)?|edge):\/\/.*/)
```

Comment:
I'd maybe drop the non-capture group flags `:?` here, since we never use capture groups here anyway and it makes the expressions a bit more confusing. Something like this should suffice:

```
/(chrome(-extension)?|edge):\/\/.*/
```

Reply:
I agree! Fixed in 0353d3b94452b0a508a2df9d2095adaf7502304b.